pkgs/tools/misc/exa/default.nix

```
@@ -52,6 +52,6 @@ buildRustPackage rec {
     '';
     homepage = https://the.exa.website;
     license = licenses.mit;
-    maintainers = [ maintainers.ehegnes ];
+    maintainers = [ maintainers.ehegnes maintainers.lilyball ];
```

nitpick: 
```suggestion
    maintainers = with maintainers; [ ehegnes lilyball ];
```
I don't feel strongly about it, just a little easier for me to parse :)

---

Good point, I'll do that.

(I wish GitHub offered an "Amend" option with the built-in suggestion feature)

---

I think nixpkgs is a special case where we care a lot about what gets put into master

---

And at work for example, all PRs are "squashed" to a generic "PR Merged #xxxx: " commit

Whether or not that's great, idk

---

Yeah, I mean I get that GitHub probably doesn't want to expose a history-editing feature, but still, even ignoring Nixpkgs's commit guidelines I prefer to curate my history.

I hate squash merges. History is important.

---

I think at work, it's because no one wants to use git on the command line, but also not have a bunch of micro-commits. So like all good compromises, we selected something that kinda crap :)